src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,50 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+
+    @Test
+    @DisplayName("문자열 '1,2'를 ','로 분리하여 두 개의 숫자로 나누어 결과 확인")
+    void splitTwoNum() {
+        String[] result = "1,2".split(",");
+        assertThat(result).containsExactly("1", "2");
+    }
+
+    @Test
+    @DisplayName("문자열 '1'을 ','로 분리하여 하나의 숫자를 포함하는 배열 확인")
+    void splitOneNum() {
+        String[] result = "1".split(",");
+        assertThat(result).containsExactly("1");
+    }
+
+    @Test
+    @DisplayName("substring() 메서드로 문자열 '(1,2)'에서 괄호를 제거하고 내용 추출")
+    void substringParentheses() {
+        String result = "(1,2)".substring(1, 4);
+        assertThat(result).isEqualTo("1,2");
+    }
+
+    @Test
+    @DisplayName("charAt() 메서드를 사용하여 특정 위치의 문자 추출")
+    void getCharacterUsingCharAt() {
+        String str = "abc";
+        char result = str.charAt(0);
+        assertThat(result).isEqualTo('a');
+    }
+
+    @Test
+    @DisplayName("인덱스가 범위를 벗어날 경우 예외 발생 확인")
+    void StringIndexOutOfBoundsExceptionTest() {
```

Comment:
메서드의 이름은 소문자로 시작하도록 작성해보면 어떨까요?
```suggestion
    void stringIndexOutOfBoundsExceptionTest() {
```